extensions/resource-deployment/src/ui/notebookWizard/notebookWizard.ts

```diff
@@ -98,8 +104,8 @@ export class NotebookWizard extends WizardBase<NotebookWizard, NotebookWizardPag
 		return notebook;
 	}
 
-	private getPages(): NotebookWizardPage[] {
-		const pages: NotebookWizardPage[] = [];
+	private getPages(): ResourceTypePage[] {
+		const pages: ResourceTypePage[] = [];
```

Comment:
Since NotebookWizardPage derives from ResoruceTypePage getPages can still retur NotebookWizardPage[] right?

Reply:
Yes, it will work fine.